app/partials/showModuleNewRelease.html

```diff
@@ -10,6 +10,8 @@
 <div class="row">
 
   <div class="jumbotron upload-jumbotron" ng-controller="ReleaseFileCtrl"
+  ui-view="upload_new_release"
```

Comment:
@elliottwilliams, it doesn't. Through trial and error I found that I could only make autoscroll work if the element has a ui-view attribute. Do you know of a way to make autoscroll work without faking view IDs?